views/aggregate-flow-metadata.json

```diff
@@ -2014,10 +2042,14 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-dbt",
             "description": {
                 "summary": "Triggers a dbt Cloud job run and waits for thetriggered run to complete."
-            }
+            },
+            "documentation_url": "https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs",
+            "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/9a001902ed43a84c6c96d23b24622e19/dbt-bit_tm.png?h=250",
+            "install_command": "pip install prefect-dbt",
+            "location_in_collection": "prefect_dbt.cloud.jobs",
+            "collection_repo_url": "https://github.com/PrefectHQ/prefect-dbt"
         },
         "Wait for dbt Cloud job run": {
             "name": "Wait for dbt Cloud job run",
```

Comment:
Should we also include a slug of the function name?

Reply:
that is the value for the `name` key

Reply:
Does this flow metadata include examples or how to import?

Reply:
the `name` key gives `flow.name` but whether its a slug or human readable name depends on whether the implementor passed a `name` kwarg to `@flow` - so ideas on how to resolve that would be welcome

>Does this flow metadata include examples or how to import?

yes, in the examples found in description, just pulled from the docstring

Reply:
```
im>>> from prefect import flow

@flow>>>
>>> @flow(name="my test flow")
... def test_flow():
...     print("hey")
...
>>> test_flow.__name__
'test_flow'
```

Reply:
added